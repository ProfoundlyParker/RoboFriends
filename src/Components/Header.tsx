import React, { Component } from "react";

class Header extends Component {
    // Will never re-render header
    shouldComponentUpdate(nextProps: any, nextState: any): boolean {
        return false;
    }
    render(): JSX.Element {
        return <h1 className="f1">RoboFriends</h1>
    }
    
}

export default Header;