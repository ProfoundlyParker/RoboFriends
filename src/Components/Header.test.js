import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

//Snapshot
it("expect to snapshot Header component", () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
