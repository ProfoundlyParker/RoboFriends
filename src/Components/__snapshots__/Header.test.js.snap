// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to snapshot Header component 1`] = `
<h1
  className="text-5xl mb-9 mt-9"
>
  RoboFriends
</h1>
`;
